lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -70,14 +82,14 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
 
     /** Toggles whether to display empty values in the card view */
     @Input()
-    displayEmpty: boolean = false;
+    displayEmpty = false;
```

Comment:
The documentation for this component needs to be updated with the new properties

Reply:
After looking into it, the documentation file for ContentMetadataComponent does not exist. I got it confused with the ContentMetadataCardComponent documentation. So you can skip this comment. Thanks!